digdag-core/src/main/java/io/digdag/core/database/DatabaseSessionStoreManager.java

```diff
@@ -288,6 +287,38 @@ public boolean isAnyNotDoneAttempts()
     }
 
     @Override
+    public List<StoredSessionAttempt> findAttemptsCreatedBeforeWithState(Instant createdBefore, int state, long lastId, int limit)
```

Comment:
I guess even if we use concurrent index creation, digdag will still wait for the index creation operation to complete before bringing up undertow and serving api requests?

If we worry about this index creation taking a long time, perhaps it would be better to do it manually before deploying this PR?

Perhaps we should consider separating database schema migration into a separate deployment step?

I.e.

0. Merge PR
1. Run database schema migration
2. Release & Deploy

As database schema migrations must anyway be backwards compatible in order to avoid digdag downtime, performing the schema migration before deploying the new digdag version should be safe.

Thoughts?